test/e2e_node/swap_test.go

```diff
@@ -65,6 +69,33 @@ var _ = SIGDescribe("Swap", "[LinuxOnly]", nodefeature.Swap, framework.WithSeria
 		gomega.Expect(isSwapFeatureGateEnabled()).To(gomega.BeTrueBecause("NodeSwap feature should be on"))
 	})
 
+	ginkgo.BeforeEach(func() {
+		ginkgo.By("Checking that swap is available on the node")
+		isSwapEnabled, err := isSwapEnabledOnNode()
+		framework.ExpectNoError(err)
+		gomega.Expect(isSwapEnabled).To(gomega.BeTrueBecause("swap should be enabled on the node"))
+
+		ginkgo.By("Resetting the swap")
+		err = resetSwap()
+		framework.ExpectNoError(err)
+
+		ginkgo.By("Re-checking that swap is available on the node after swap reset")
+		isSwapEnabled, err = isSwapEnabledOnNode()
+		framework.ExpectNoError(err)
+		gomega.Expect(isSwapEnabled).To(gomega.BeTrueBecause("swap should be enabled on the node after reset"))
+
+		ginkgo.By("Checking that swap usage is near zero")
+		usedSwapBytes, err := getUsedSwapOnNode()
+		framework.ExpectNoError(err)
+		gomega.Expect(usedSwapBytes).To(gomega.BeNumerically("<", 200*1024*1024), "swap usage after reset should be less than 200Mi")
+
+		framework.Logf("Swap is reset. Used swap bytes (should be near 0): %d", usedSwapBytes)
+
+		secondsToSleep := 10
```

Comment:
without thinking too deeply about it,10 seems pretty high for this. I assume you workshopped it and that's as low as it works reliably?

Reply:
After testing in https://github.com/kubernetes/kubernetes/pull/129634, reduced it to 3 seconds.